File: fix_store.py
with open('./lib/data/store/main/main_store.g.dart', 'r') as f:
    data = f.read()
    data = data.replace('List<dynamic> get websiteList', 'List<WebsiteTableData> get websiteList')
    data = data.replace('set websiteList(List<dynamic> value)', 'set websiteList(List<WebsiteTableData> value)')
    data = data.replace('dynamic get websiteEntity', 'WebsiteTableData? get websiteEntity')

with open('./lib/data/store/main/main_store.g.dart', 'w') as f:
    f.write(data)

print('success')

File: fix_store.dart
// https://github.com/mobxjs/mobx.dart/issues/405
import 'dart:io';

final replace = <Map<String, String>>[
  {
    'raw': 'List<dynamic> get websiteList',
    'new': 'List<WebsiteTableData> get websiteList'
  },
  {
    'raw': 'set websiteList(List<dynamic> value)',
    'new': 'set websiteList(List<WebsiteTableData> value)'
  },
  {
    'raw': 'dynamic get websiteEntity',
    'new': 'WebsiteTableData? get websiteEntity'
  },
];

void main() {
  final file = File('./lib/data/store/main/main_store.g.dart');
  var fileString = file.readAsStringSync();
  for (final re in replace) {
    fileString = fileString.replaceAll(re['raw']!, re['new']!);
  }
  file.writeAsStringSync(fileString);
  print('replace dynamic success');
}
